protocol/shannon/relay.go

```diff
@@ -44,11 +46,21 @@ func sendHttpRelay(
 	//
 	// Best practice in Go is to use a custom HTTP client Transport.
 	// See: https://vishnubharathi.codes/blog/know-when-to-break-up-with-go-http-defaultclient/
-	relayHTTPResponse, err := http.DefaultClient.Do(relayHTTPRequest)
+	client := &http.Client{
+		Timeout: timeout,
```

Comment:
This looks redundant: the HTTP request has a timeout set on it:
https://github.com/buildwithgrove/path/blob/69ea48321885e09095984302c2dbea5f8a30b20a/protocol/shannon/relay.go#L31-L32